### PR TITLE
borg spraycans have a five second delay before being able to knock someone down again

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
infinite charges = they can repeatedly apply a **6** second knockdown

~~it's nine because this is three seconds where the knockdown isnt being applied~~
it's five because kevin told me to make it five and he understands these things better than me

## Why It's Good For The Game
this is not good balance

## Changelog
:cl:
balance: borg spraycans have a five second delay before being able to knock someone down again
/:cl:

### CODE REVIEW

**Comment by silicons** [comment]
just do 5 seconds and don't set it to = -90 just set it to = 0

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -771,6 +775,7 @@
 	name = "cyborg spraycan"
 	desc = "A metallic container containing shiny synthesised paint."
 	charges = -1
+	stun_delay = 50 // 3 seconds longer than the knockdown time
```
Comment (Linzolle) [inline code/game/objects/items/crayons.dm]: i know its not exactly important but isn't this comment inaccurate now

Comment (timothyteakettle) [reply code/game/objects/items/crayons.dm]: woops thanks

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -771,6 +775,7 @@
 	name = "cyborg spraycan"
 	desc = "A metallic container containing shiny synthesised paint."
 	charges = -1
+	stun_delay = 50
```
Comment (DeltaFire15) [inline code/game/objects/items/crayons.dm]: SECONDS macro for easier readibility, good to merge other than that.
```suggestion
	stun_delay = 5 SECONDS
```

**Review by DeltaFire15 — CHANGES_REQUESTED** [review]

**Review by DeltaFire15 — APPROVED** [review]

**Pull request merged**